File: ezSCORES/UI/ezscores_desktop/lib/models/DTOs/teamDto.dart
import 'package:ezscores_desktop/models/DTOs/playerDto.dart';
import 'package:json_annotation/json_annotation.dart';

part 'teamDto.g.dart';

@JsonSerializable(explicitToJson: true)
class TeamDTO {
	int? id;
	String? name;
  List<PlayerDTO>? players;

	TeamDTO({this.id, this.name});

	factory TeamDTO.fromJson(Map<String, dynamic> json) => _$TeamDTOFromJson(json);

	Map<String, dynamic> toJson() => _$TeamDTOToJson(this);
}

File: ezSCORES/ezSCORES.Model/DTOs/TeamDTO.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ezSCORES.Model.DTOs
{
	public class TeamDTO
	{
		public int Id { get; set; }
		public string Name { get; set; }
		public byte[]? Picture { get; set; }
		public List<PlayerDTO> Players { get; set; }
	}
}
